lib/seed/facility_seeder.rb

```diff
@@ -61,15 +69,22 @@ def call
       facility_attrs = []
       fg_result.results.each do |row|
         facility_group_id, facility_group_name = *row
-        number_of_facilities_per_facility_group.times {
+        number_facilities = number_of_facilities_per_facility_group
+        state = Seed::FakeNames.instance.state
+        blocks = Seed::FakeNames.instance.blocks.sample(facilities_per_block)
+        number_facilities.times {
           size = weighted_facility_size_sample
           type = SIZES_TO_TYPE.fetch(size).sample
 
+          # TODO set the facility state here to match the parent district!
```

Comment:
Are you confused about something around the modeling here or is this just a comment to come back to?